pytmc/bin/pragmalint.py

```diff
@@ -14,16 +14,19 @@
 from .. import parser
 from . import util
 from .db import LinterError
-
+from ..pragmas import _FLEX_TERM_REGEX
 
 DESCRIPTION = __doc__
 logger = logging.getLogger(__name__)
 
 PRAGMA_START_RE = re.compile('{attribute')
 PRAGMA_RE = re.compile(
-    r"^{\s*attribute[ \t]+'pytmc'[ \t]*:=[ \t]*'([\s\S]*)'}$",
+    r"^{\s*attribute[ \t]+'pytmc'[ \t]*:=[ \t]*'(?P<setting>[\s\S]*)'}$",
     re.MULTILINE
 )
+PRAGMA_LINE_RE = re.compile(r"(" + _FLEX_TERM_REGEX + ")", re.MULTILINE)
+PRAGMA_SETTING_RE = re.compile(r"^\s*(?P<title>[a-zA-z0-9]+):(?P<setting>.*)$")
+PRAGMA_PV_LINE_RE = re.compile(r"pv\S*:")
```

Comment:
```suggestion
PRAGMA_PV_LINE_RE = re.compile(r"pv\s*:")
```
Lower-case, as it should be whitespace there